nests.json

```diff
@@ -54,6 +54,28 @@
       "status": "stable",
       "category": "widget",
       "readme": "https://raw.githubusercontent.com/ismail-yilmaz/MessageCtrl/main/README.md"
+    },
+    {
+      "name": "SurfaceCtrl",
+      "packages": [
+        "SurfaceCtrl"
+      ],
+      "description": "A 3D viewer few different 3D object data format",
+      "repository": "https://github.com/Xemuth/SurfaceCtrl.git",
+      "status": "beta",
```

Comment:
@ismail-yilmaz could you design which statuses can be set. I think we have experimental which is very similar to beta. We need to create unified protocol.

Reply:
@klugier 

Well, the original scheme used in bazaar (the traditional way) was good and familiar enough, IMO. But here we can use:

- `experimental`: Package is new, working but it is not expected to be stable or considered finished.
- `stable`: Package has a stable api and is well tested.
- `rolling`: This means that the package is well tested and stable in the sense that it has a defined behaviour, not crash-prone, but adds new stuff frequently or occasionally (e.g. TerminalCtrl)
- `deprecated`: The package is deprecated, has a better alternative, but possibly required by exisiting apps or other packages.
- `obsolete`: Package is not used anymore. Provided as an archive item.

What do you think? We can change, expand, refine these "scratch" definitions.

Reply:
@ismail-yilmaz  I am fine with that scheme - please add it to the README.md. Thanks for providing this list.

Reply:
@klugier 

Sure, but I believe he already stated his opinion below (If I am not interpreting wrongly - the 'broken' category is his idea too. ). :)

Reply:
I didn't scroll that far :)

Reply:
@klugier 

No problem.  I am now resolving this conversation and I'll make the relevant changes to the README file. and send a pull request, later tonight.